app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/EmailPasswordSignInSignUpFragment.kt

```diff
@@ -115,6 +115,10 @@ class EmailPasswordSignInSignUpFragment : Fragment() {
                 is SignInResult.CodeRequired -> {
                     displayDialog(message = getString(R.string.sign_in_switch_to_otp_message))
                 }
+                is SignInResult.MFARequired -> {
+                    // Please refer to the MFA Fragment for handling MFA branches if conditional access - MFA is enabled. MFA is under private preview.
```

Comment:
I'd omit to mention that MFA is under private preview. So, we will not need to update the sample once we move to public preview
```suggestion
                    // Please refer to the MFA Fragment for handling MFA branches if conditional access - MFA is enabled.
```